pkg/api/controller.go

```diff
@@ -52,31 +51,8 @@ func (c *Controller) Run() error {
 	}
 
 	// Updating the CVE Database
```

Comment:
Remove this comment
And also add another comment which says,
// Enable extensions based on configuration

Reply:
Ok